src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadHandler.java

```diff
@@ -38,4 +52,57 @@ public List<CaseEvent> handledEvents() {
         return EVENTS;
     }
 
+    private CallbackResponse validateValues(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        List<String> errors = new ArrayList<>();
+
+        checkDateCorrectness(errors, caseData.getDocumentUploadWitness1(), date -> date.getValue().getWitnessOption1UploadDate(),
+                             "Invalid date: witness statement date entered must not be in the future (1).");
```

Comment:
Looks like I forgot to go on a new line when we refactored this code together.

Please put the error message on this and the following statements on a new line (and modify them in the way you think is more descriptive), so that they don't break the checkstyle rules.